classes/PodsData.php

```diff
@@ -1833,12 +1826,7 @@ public static function table_create( $table, $fields, $if_not_exists = false ) {
 	 */
 	public static function table_alter( $table, $changes ) {
 
-		/**
-		 * @var $wpdb wpdb
-		 */
-		global $wpdb;
-
-		$sql = "ALTER TABLE `{$wpdb->prefix}" . self::$prefix . "{$table}` {$changes}";
+		$sql = "ALTER TABLE `" . self::get_pods_prefix() . "{$table}` {$changes}";
```

Comment:
:no_entry_sign: String "ALTER TABLE `" does not require double quotes; use single quotes instead

<sub>hash: 1ee7aa9b90e9a8e49f9638cfb9e17690</sub>